interceptors/logging/payload_test.go

```diff
@@ -60,7 +60,6 @@ func TestPayloadSuite(t *testing.T) {
 func (s *loggingPayloadSuite) TestPing_LogsBothRequestAndResponse() {
 	_, err := s.Client.Ping(s.SimpleCtx(), goodPing)
 	require.NoError(s.T(), err, "there must be not be an error on a successful call")
-
```

Comment:
Was this intentional? I'd prefer to keep this newline

Reply:
Sure